netbout-web/src/test/java/com/netbout/dynamo/DyAttachmentsITCase.java

```diff
@@ -74,4 +74,35 @@ public void createsAndLoadsAttachments() throws Exception {
         attachments.delete(name);
     }
 
+    /**
+     * DyAttachments can create, save and load attachments.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void throwsExceptionForNameLength() throws Exception {
+        final String alias = "bill1";
+        final Aliases aliases =
+            new DyBase().user(new URN("urn:test:840919")).aliases();
+        aliases.add(alias);
+        final Inbox inbox = aliases.iterate().iterator().next().inbox();
+        final Bout bout = inbox.bout(inbox.start());
+        final Attachments attachments = bout.attachments();
+        final String name = "a";
+        try {
+            attachments.create(name);
+        } catch (final Attachments.InvalidNameException ex) {
```

Comment:
@kujtimiihoxha you should also make sure that the exception will be thrown, e.g. by adding an `Assert.fail(...)` in the try.